magenta/models/music_vae/README.md

```diff
@@ -41,7 +41,6 @@ MusicVAE models in the browser. Existing applications built with this library in
 Learn more about the API in its [repo](https://goo.gl/magenta/musicvae-js).
```

Comment:
Not sure how you ended up with changes to this file but can you revert them?  Looks good other than that!

Reply:
Done.